bookdown/00-intro.Rmd

```diff
@@ -16,11 +16,10 @@ It is intended for users who have basic knowledge in machine learning and R and
 
 `r mlr_pkg("mlr")` was first released to CRAN in 2013, with the core design and architecture dating back much further.
 Over time, the addition of many features has led to a considerably more complex design that made it harder to build, maintain, and extend than we had hoped for.
-<!-- data.table v.1.0 was released in 2006, see https://github.com/Rdatatable/data.table/wiki-->
-Further, the R ecosystem was not as far developed at the time, for example `r cran_pkg("data.table")` did not exist.
-Including such helpful packages makes `r mlr_pkg("mlr3")` easier to maintain, to use, and more performant, but also resulted in non-trivial API changes that would have been hard to implement and support in `r mlr_pkg("mlr")`.
-With hindsight, we saw that some of the design and architecture changes in `r mlr_pkg("mlr")` made it difficult to support new features, in particular with respect to pipelines.
-In 2018, we decided to start working on the reimplementation, and in July 2019 we released the first version of `r mlr_pkg("mlr3")` to CRAN.
+Furthermore, the R ecosystem as well as helpful packages such as `r cran_pkg("data.table")` have undergone major changes in the meantime.
```

Comment:
@pat-s pls have a look if you like my rewrite and if it expresses what you wanted to say